classes/Tools.php

```diff
@@ -2751,6 +2751,20 @@ public static function clearSmartyCache()
         Tools::clearCompile($smarty);
     }
 
+    /**
+     * Clear Symfony cache
+     */
+    public static function clearSf2Cache($env = null)
+    {
+        if (!$env) {
+            $env = _PS_MODE_DEV_ ? 'dev' : 'prod';
+        }
+
+        $sf2Refresh = new \PrestaShopBundle\Service\Cache\Refresh($env);
```

Comment:
As you can see in this pull request it is already done in a lot of places. I just DRYed it.